packages/traefik/generated-changes/patch/values.yaml.patch

```diff
@@ -1,8 +1,10 @@
 --- charts-original/values.yaml
 +++ charts/values.yaml
-@@ -3,9 +3,9 @@
-   # -- Traefik image host registry
-   registry: docker.io
```

Comment:
Just to be clear, we're dropping support for `Values.image.registry` because we prepend the system-default-registry to the image `repositoriy` value, and having `docker.io` added before it here breaks that.